### PR TITLE
feat: Removed delete workflow from CD

### PR DESCRIPTION
**Pull request by vivek-devtron**
# Description

User already has a CI running in some platform (Jenkins, Circle CI etc.) and they want to use Devtron for deployment by leveraging their existing CI

Fixes # [AB#936](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/936)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?
- verified by creating new webhook CD
- deletion of webhook CD
- validated current flow working

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by sohel-rp — APPROVED** [review]

File: src/components/ciPipeline/Webhook/WebhookDetailsModal.tsx [hunk 1]
```diff
@@ -57,6 +57,7 @@ export function WebhookDetailsModal({ close }: WebhookDetailType) {
     const [isSuperAdmin, setIsSuperAdmin] = useState(false)
     const [samplePayload, setSamplePayload] = useState<any>(null)
     const [modifiedSamplePayload, setModifiedSamplePayload] = useState<any>(null)
+    const [modifiedSampleString, setModifiedSampleString] = useState<string>(null)
```
Comment (shivani170) [inline src/components/ciPipeline/Webhook/WebhookDetailsModal.tsx:60]: Why are we initializing string with a null value?

File: src/components/ciPipeline/Webhook/WebhookDetailsModal.tsx [hunk 2]
```diff
@@ -129,12 +130,20 @@ export function WebhookDetailsModal({ close }: WebhookDetailType) {
...
+                _modifiedPayload.dockerImage = ''
+            }
             delete _modifiedPayload.ciProjectDetails
             const modifiedJSONString = formatSampleJson(_modifiedPayload)
             setSamplePayload(_modifiedPayload)
             setModifiedSamplePayload(_modifiedPayload)
+            setModifiedSampleString(modifiedJSONString)
             setSampleJSON(modifiedJSONString)
-            setSampleCURL(CURL_PREFIX + modifiedJSONString)
+            setSampleCURL(
```
Comment (shivani170) [inline src/components/ciPipeline/Webhook/WebhookDetailsModal.tsx:142]: could you please add a comment what are we doing here

File: src/css/base.scss [hunk 3]
```diff
@@ -2282,3 +2282,7 @@ textarea::placeholder {
     left: 50%;
     transform: translate(-50%, -50%);
 }
+
```
Comment (shivani170) [inline src/css/base.scss:2285]: We already have this classname in base.scss

Comment (vivek-devtron) [reply src/css/base.scss:2285]: done

**Review by shivani170 — APPROVED** [review]
Looks Good

**Review by sohel-rp — APPROVED** [review]

**Pull request merged**